doc/_sphinx/styles/custom.css

```diff
@@ -146,3 +146,8 @@ main.bd-content #main-content .prev-next-bottom .right-next .prevnext-label, mai
 .bd-sidebar .nav li>a {
     padding: .1rem 1.5rem;
 }
+
+img.logo {
+  width: 100% !important;
```

Comment:
I had to use important here, otherwise it would not work... not sure if there is a better way

Reply:
You can change line 55 (`max-width: 24vh;`) into `width: 100%` instead. Then you won't need the "important" override.

Reply:
it worked!